htmlbook-xsl/xspec/fileextension.xspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<x:description xmlns:x="http://www.jenitennison.com/xslt/xspec"
+               xmlns:functx="http://www.functx.com"
+	       xmlns="http://www.w3.org/1999/xhtml"
+	       xmlns:ncx="http://www.daisy.org/z3986/2005/ncx/"
+	       xmlns:h="http://www.w3.org/1999/xhtml"
+	       xmlns:e="http://github.com/oreillymedia/epubrenderer"
+	       xmlns:htmlbook="https://github.com/oreillymedia/HTMLBook"
+               stylesheet="../chunk.xsl">
+
+<x:scenario label="When calling output-filename-for-chunk" pending="We received an  error when trying to run this scenario.
```

Comment:
Is there an extra space here between `an` and `error`?

Reply:
also, can you put a newline and two spaces before `label` and then another another newline and two spaces before `pending`, to make the lines even shorter?